### ISSUE TITLE
[Chatllama] actor.py: the load function  is correct?

### ISSUE THREAD

**Issue by bucm-tcm-tool** [nebuly-ai/nebullvm#274]
**Here are some possible errors:**

- **The code in the rlhf/actor.py, line 117:** 

`      self.head.load_state_dict(model_dict["head"])`
We have never defined "self.head", so should we delete this line ?

- **The code in the rlhf/actor.py, line 133-136:** 

`      torch.save(
           {"model": self.model.state_dict(), "head": self.head.state_dict()},
            path,
        )`

should we delete "head": self.head.state_dict() ?

**Comment by PierpaoloSorbellini** [comment]
Hi @bucm-tcm-tool, thanks for the catch.
Yes, the head is not needed, it was a typo from a copy-paste of the reward load/save. 
It should be fixed by now.
